Sources/NIOFileSystem/BufferedWriter.swift

```diff
@@ -186,6 +186,25 @@ extension WritableFileHandleProtocol {
             capacity: Int(capacity.bytes)
         )
     }
+    
+    /// Convenience function that creates a buffered reader, executes
+    /// the closure that writes the contents into the buffer and calls 'flush()'.
+    ///
+    /// - Parameters:
+    ///   - body: The closure that writes the contents to the buffer created in this method.
+    ///   - Returns: The result of the executed closure.
```

Comment:
`Returns:` shouldn't be indented here, otherwise it's treated as documentation for a parameter